src/components/CommunityEvents/index.tsx

```diff
@@ -96,13 +100,13 @@ const CommunityEvents = () => {
       <Center w={{ base: "100%", lg: "40%" }}>
         <Box pe={8} ps={{ base: 8, lg: 0 }}>
           <OldHeading>
-            <Translation id="community-events-content-heading" />
+            <Translation id="page-index:community-events-content-heading" />
```

Comment:
can use hook directly

```suggestion
{t("page-index:community-events-content-heading")}
```